common/odigos_config.go

```diff
@@ -1,5 +1,241 @@
+		consts.OpenshiftEnabledProperty:           ConfigBool(config.OpenshiftEnabled),
+		consts.PspProperty:                        ConfigBool(config.Psp),
+		consts.SkipWebhookIssuerCreationProperty:  ConfigBool(config.SkipWebhookIssuerCreation),
+		consts.AllowConcurrentAgentsProperty:      ConfigBoolPointer{Value: config.AllowConcurrentAgents},
+		consts.ImagePrefixProperty:                ConfigString(config.ImagePrefix),
+		consts.UiModeProperty:                     ConfigString(config.UiMode),
+		consts.UiPaginationLimitProperty:          ConfigInt(config.UiPaginationLimit),
+		consts.UiRemoteUrlProperty:                ConfigString(config.UiRemoteUrl),
+		consts.CentralBackendURLProperty:          ConfigString(config.CentralBackendURL),
+		consts.ClusterNameProperty:                ConfigString(config.ClusterName),
+		consts.IgnoredNamespacesProperty:          ConfigStringList(config.IgnoredNamespaces),
+		consts.IgnoredContainersProperty:          ConfigStringList(config.IgnoredContainers),
+		consts.MountMethodProperty:                (*ConfigMountMethod)(config.MountMethod),
+		consts.CustomContainerRuntimeSocketPath:   ConfigString(config.CustomContainerRuntimeSocketPath),
+		consts.K8sNodeLogsDirectory:               (*ConfigCollectorNode)(config.CollectorNode),
+		consts.UserInstrumentationEnvsProperty:    (*ConfigUserEnv)(config.UserInstrumentationEnvs),
+		consts.AgentEnvVarsInjectionMethod:        (*ConfigEnvInjection)(config.AgentEnvVarsInjectionMethod),
+		consts.NodeSelectorProperty:               ConfigNodeSelector(config.NodeSelector),
+		consts.KarpenterEnabledProperty:           ConfigBoolPointer{Value: config.KarpenterEnabled},
+		consts.RollbackDisabledProperty:           ConfigBoolPointer{Value: config.RollbackDisabled},
+		consts.RollbackGraceTimeProperty:          ConfigString(config.RollbackGraceTime),
+		consts.RollbackStabilityWindow:            ConfigString(config.RollbackStabilityWindow),
+		consts.AutomaticRolloutDisabledProperty:   (*ConfigRollout)(config.Rollout),
+		consts.OidcTenantUrlProperty:              (*ConfigOidcTenant)(config.Oidc),
+		consts.OidcClientIdProperty:               (*ConfigOidcClientId)(config.Oidc),
+		consts.OidcClientSecretProperty:           (*ConfigOidcClientSecret)(config.Oidc),
+		consts.OdigletHealthProbeBindPortProperty: ConfigInt(config.OdigletHealthProbeBindPort),
+		consts.ServiceGraphDisabledProperty:       (*ConfigCollectorGateway)(config.CollectorGateway),
+		consts.GoAutoOffsetsCronProperty:          ConfigString(config.GoAutoOffsetsCron),
+		consts.ClickhouseJsonTypeEnabledProperty:  ConfigBoolPointer{Value: config.ClickhouseJsonTypeEnabledProperty},
+	}
+
+	return displayData
+}
+
+func PrintMap(config *OdigosConfiguration) {
+	display := makeAMap(config)
+
+	var order []string
+	for k := range consts.ConfigDisplay {
+		order = append(order, k)
+	}
+
+	sort.Strings(order)
+
+	for _, key := range order {
+		fmt.Printf("	- %s", key)
```

Comment:
```suggestion
		fmt.Printf("	- %s: ", key)
```

add the colon+space here so you only need to write it once

Reply:
fixed